frontend/src/userstories/SignIn.test.js

```diff
@@ -16,9 +16,25 @@ describe('Sign in integration-test:', () => {
     const game = { id: 'gameId', players: [{ name: playerName }] };
     //then
     axios.get = jest.fn(
```

Comment:
Check out [axios-mock-adapter](https://www.npmjs.com/package/axios-mock-adapter) for ease of use

Reply:
that should indeed make  life easier! thx